apps/docs/package.json

```diff
@@ -20,6 +20,10 @@
         "build:previewRef": "tsx scripts/generatePreviewRef.ts",
         "copy:images": "tsx scripts/copyImages.ts"
     },
+    "peerDependencies": {
+        "react-aria": "^3.34",
+        "react-aria-components": "^1.3.0"
```

Comment:
I've updated it in my pr, but I noticed that we also have react-aria-components inside dependencies that is not the latest version.

Reply:
Yes I don't really understand why peers are at 1.3 but not everywhere else.

Reply:
I guess I updated badly.. sorry :/ I don't know how I didn't see them before.

Reply:
so don't be afraid to put things at their latest version